config/prod/neo4j2owl-config.yaml

```diff
@@ -211,6 +211,7 @@ neo_node_labelling:
     label: diaphragm
   - classes:
       - UBERON:0010000
+      - UBERON:0000178
```

Comment:
Is this enough to add the tag to subClasses as well or do we need some DL query in here?

Reply:
We had previous examples, that I think are also meant to pull in subclasses. E.g.,

```
  - classes:
      - HANCESTRO:0004
    label: Race
```

Will ask @dosumis to confirm.

Reply:
I guess you are right, that would also add the tag to the subclasses as well given the 'Race' example. I wasn't sure, thanks for the other example

Reply:
A single class is a DL query - just with a named class.